test/testUtils/ConsignmentStatusOptions.scala

```diff
@@ -0,0 +1,72 @@
+    ("client checks failed", CurrentStatus(completed, completed, completed, failed, None, None), "/file-checks-results", "Failed", "View errors"),
+    ("client checks completed with issues", CurrentStatus(completed, completed, completed, completedWithIssues, None, None), "/file-checks-results", "Failed", "View errors"),
+    ("client checks completed", CurrentStatus(completed, completed, completed, completed, None, None), "/file-checks-results", "In Progress", "Resume transfer"),
+    ("confirm transfer completed", CurrentStatus(completed, completed, completed, completed, completed, None), "/file-checks-results", "In Progress", "Resume transfer"),
+    (
+      "export in progress",
+      CurrentStatus(completed, completed, completed, completed, completed, inProgress),
+      "/additional-metadata/download-metadata/csv",
+      "In Progress",
+      "Download report"
+    ),
+    (
+      "export failed",
+      CurrentStatus(completed, completed, completed, completed, completed, failed),
+      "mailto:nationalArchives.email?subject=Ref: consignment-ref-1 - Export failure",
+      "Failed",
+      "Contact us"
+    ),
+    (
+      "export completed",
+      CurrentStatus(completed, completed, completed, completed, completed, completed),
+      "/additional-metadata/download-metadata/csv",
+      "Transferred",
+      "Download report"
+    )
+  )
+
+  val expectedJudgmentStatesAndStatuses: TableFor5[String, CurrentStatus, String, String, String] = Table(
+    ("expected transfer state", "current status", "action url", "transfer state", "action text"),
+    ("before upload", CurrentStatus(None, None, None, None, None, None), "/before-uploading", "In Progress", "Resume transfer"),
+    ("upload in progress", CurrentStatus(None, None, inProgress, None, None, None), "/upload", "In Progress", "Resume transfer"),
+    ("upload failed", CurrentStatus(None, None, failed, None, None, None), "/upload", "Failed", "View errors"),
+    ("upload completed", CurrentStatus(None, None, completed, None, None, None), "/upload", "In Progress", "Resume transfer"),
+    ("client checks in progress", CurrentStatus(None, None, completed, inProgress, None, None), "/file-checks", "In Progress", "Resume transfer"),
+    ("client checks failed", CurrentStatus(None, None, completed, failed, None, None), "/file-checks-results", "Failed", "View errors"),
+    ("client checks completed with issues", CurrentStatus(None, None, completed, completedWithIssues, None, None), "/file-checks-results", "Failed", "View errors"),
+    ("client checks completed", CurrentStatus(None, None, completed, completed, None, None), "/file-checks-results", "In Progress", "Resume transfer"),
+    ("confirm transfer completed", CurrentStatus(None, None, completed, completed, None, None), "/file-checks-results", "In Progress", "Resume transfer"),
```

Comment:
The "confirm transfer completed" state looks the same as the "client checks completed" test. Did you mean to do this instead `CurrentStatus(completed, completed, completed, completed, completed, None)`

Reply:
Good spot